docs/features/overview.rst

```diff
@@ -394,18 +394,27 @@ model. This model outputs empty string in case if there is no answer in context.
 Morphological tagging model :doc:`[docs] </features/models/morphotagger>`
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Based on character-based approach to morphological tagging `Heigold et al., 2017. An extensive empirical evaluation of
-character-based morphological tagging for 14 languages <http://www.aclweb.org/anthology/E17-1048>`__. A state-of-the-art
-model for Russian and several other languages. Model takes as input tokenized sentences and outputs the corresponding
-sequence of morphological labels in `UD format <http://universaldependencies.org/format.html>`__. The table below
-contains word and sentence accuracy on UD2.0 datasets. For more scores see :doc:`full table </features/models/morphotagger>`.
+We have a BERT-based model for Russian and character-based models for 11 languages.
+The character model is based on `Heigold et al., 2017. An extensive empirical evaluation of
+character-based morphological tagging for 14 languages <http://www.aclweb.org/anthology/E17-1048>`__.
+It is a state-of-the-art model for Russian and near state of the art for several other languages.
+Model takes as input tokenized sentences and outputs the corresponding
+sequence of morphological labels in `UD format <http://universaldependencies.org/format.html>`__.
+The table below contains word and sentence accuracy on UD2.0 datasets.
+For more scores see :doc:`full table </features/models/morphotagger>`.
 
 .. table::
     :widths: auto
 
     +----------------------+--------------------------------------------------------------------------------------------------------------+---------------+----------------+--------------------+
     |    Dataset           | Model                                                                                                        | Word accuracy | Sent. accuracy | Download size (MB) |
     +======================+==============================================================================================================+===============+================+====================+
+    |`UD2.3`_ (Russian)    |`UD Pipe 2.3`_ (Straka et al., 2017)                                                                          |    93.5       |                |                    |
```

Comment:
Why there is not results for other languages (mentioned on morphotagger doc page)?

Reply:
I give only the most impressive ones since it is an overview page for the whole library, not for my own component.